azurelinuxagent/common/osutil/default.py

```diff
@@ -104,18 +105,15 @@ def _get_firewall_delete_conntrack_accept_command(wait, destination):
 
 
 def _get_firewall_delete_owner_accept_command(wait, destination, owner_uid):
-    return _add_wait(wait, ["iptables", "-t", "security", "-D", "OUTPUT", "-d", destination, "-p", "tcp", "-m", "owner",
-                            "--uid-owner", str(owner_uid), "-j", "ACCEPT"])
+    return _add_wait(wait, AddFirewallRules.get_iptables_accept_command("", "-D", destination, owner_uid))
```

Comment:
The function `get_iptables_accept_command(wait, command, destination, owner_uid)` takes in `wait` as the first parameter. You can simply use this call - 

`return AddFirewallRules.get_iptables_accept_command(wait, "-D", destination, owner_uid)`

Reply:
Additionally, could you parameterize `-D` just like you did `-I` and `-A` please (maybe even make it an Enum potentially?)

Reply:
> Additionally, could you parameterize `-D` just like you did `-I` and `-A` please (maybe even make it an Enum potentially?)

NIT on this comment

Reply:
I parameterized all the commands , am I missing any NIT comment ?